src/Transaction.jsx

```diff
@@ -7,6 +7,30 @@ const TEST_CATEGORY_ID = 'test-category-id';
 const TEST_PAYEE_ID = 'test-payee-id';
 
 export default class Transaction {
+  static fromFirebaseData(key, data) {
+    let inflow = 0.0;
+    let outflow = 0.0;
+    if (data.amountMinor < 0) {
+      outflow = Math.abs(data.amountMinor) / 100;
```

Comment:
If this conversion happens here, why is `amountMinor` still usable from `Transaction`?